language/en/units.json

```diff
@@ -1142,8 +1175,17 @@
 			"corcat": "Heavy Rocket Bot",
 			"corch": "Tech 1 Constructor",
 			"corck": "Tech 1 Constructor",
-			"corcom": "Cortex Commander",
+			"corcom": "Commander",
 			"corcomboss": "Oh s##t we are so dead",
+			"corcomlvl2": "Commander",
+			"corcomlvl3": "Specialized in frontline warfare",
+			"corcomlvl4": "Specialized in frontline warfare",
+			"corcomlvl5": "Specialized in frontline warfare",
+			"corcomlvl6": "Specialized in frontline warfare",
+			"corcomlvl7": "Specialized in frontline warfare",
+			"corcomlvl8": "Brrrrrrrrrrrrrrrr",
+			"corcomlvl9": "Brrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr",
+			"corcomlvl10": "BRRRRRRRRRRRRRRRRRRRRRRRRRRRRRR",
```

Comment:
Please avoid nonsense and in-jokes, they are difficult to translate and only make sense to those already in the know.

Reply:
OK, I'll update and commit this requested change.